app/models/nomination_email.rb

```diff
@@ -2,4 +2,14 @@
 
 class NominationEmail < ApplicationRecord
   belongs_to :school
+
+  NOMINATION_EXPIRY_TIME = 7.days
+
+  def nomination_expired?
```

Comment:
Not important but I would just call this expired? as it's clear we are talking about nominations or nomination emails

Reply:
We also need unit tests for it

Reply:
Done